src/components/Article.js

```diff
@@ -83,13 +83,11 @@ const ArticleInner = ({ lang, articleTitle, initialSubTitle }) => {
   const [currentSection, setCurrentSection, currentPage] = useArticlePagination(containerRef, article, subTitle)
   const section = article.sections[currentSection]
 
-  const goToArticleSubpage = (item) => {
-    if (item) {
-      const { sectionIndex, title } = item
-      setCurrentSection(sectionIndex)
-      setSubTitle(title)
-      route(`/article/${lang}/${articleTitle}/${title}`, true)
-    }
+  const goToArticleSubpage = item => {
```

Comment:
You can deconstruct directly in the arguments definition by using `({ sectionIndex, title })` instead of `item`